### PR TITLE
Draft post: community groups

### PR DESCRIPTION
**Pull request by k8hertweck**
If your pull request is for adding a blog post, please check that you include the following:

- [x] _Title_ - A short, catchy description to entice readers
- [x] _Author_ - Your name, which should match your contributor [file](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#adding-yourself-as-a-contributor). If you do not have a contributor file, please feel free to contact one of our team members to assist you
- [x] _Categories_ - Select 1 at minimum of either Community, Technical, or Science as [describe here](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)
- [x] _Tags_ - Suggested keywords that can help readers find information from among our blog posts. Please suggest a few tags following [our guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#tags).
- [x] _Content_ - This is the body of the blog post! We recommend relatively short posts (no more than five paragraphs), equivalent to a three minute read. Pictures and links to other sites are also great!

The Coop team member reviewing the submission will check for the following:

- [x] The post is appropriate for the blog; [see guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)

- [x] The author is added as a [contributor](https://fredhutch.github.io/coop/contributors/) to the blog

- [x] The file is in the DRAFT folder

- [x] A plan for publication (adding `YEAR-MONTH-DAY` to filename and moving to posts folder).

If the pull request is not for adding a blog then delete the content above and briefly describe below the changes proposed in the pull request.

### CODE REVIEW

File: _drafts/community-group-overview.md [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
+- **R Coding Group**: The R User Group meets weekly to discuss programming, data analysis, and troubleshooting in R. Bring your lunch, code, questions, and thoughts! R coders of all levels of expertise are welcome. Contact the Coop (coophelp at fredhutch.org) for more information. Additionally, check out the #r-user-comm channel/discussion within the [Coop Communities Slack](https://fredhutch.github.io/coop/community/slack-new/) for more information and our other training and community resources at [Training, Community and Finding Help](https://sciwiki.fredhutch.org/scicomputing/reference_training/). Note: If you do not have a laptop available to you we still want you to come!! You can email code privately ahead of each event to Amy Paguirigan (apaguiri at fredhutch.org). Current meeting schedules and locations are available on the [Google calendar](https://fredhutch.github.io/FHBig/calendar/). When & where: every Thursday from noon to 1 pm in the Coop Lab (M1-B406, Arnold).
```
Comment (k8hertweck) [inline _drafts/community-group-overview.md]: R User Group

File: _drafts/community-group-overview.md [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
 
 #### How to start one
```
Comment (k8hertweck) [inline _drafts/community-group-overview.md]: Requesting a new community group

File: _drafts/community-group-overview.md [hunk 3]
```diff
@@ -1,48 +1,55 @@
...
 
-#### What they are
+#### Current Community Groups
 
-Here are some of the current community groups that we have here at the Hutch but it's not exhaustive as you can always create a new one but following the [guidelines here](.../2010-01-07-post-overview_community_group#how-to-start-one).
+Here are some of the [current community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html) that we have here at the Hutch but it's not exhaustive as you can always create a new one by following the guidelines below.
```
Comment (k8hertweck) [inline _drafts/community-group-overview.md]: I would soften the language here: say, "Interested in starting a community group focused on a topic not listed here? See below for instructions on how to request a new group."

File: _drafts/community-group-overview.md [hunk 4]
```diff
@@ -1,48 +1,55 @@
...
 
-- **Coop community meetings**: The Coop hosts community meetings once a quarter to share information from the data science community at the Hutch. Join us to learn about projects and events happening in our community! _**When & where**: Quarterly, usually A1-303 but check calendars_
+- **Coop Community Meetings**: The Coop hosts community meetings once a quarter to share information from the data science community at the Hutch. Join us to learn about projects and events happening in our community! When & where: Quarterly, usually A1-303 but check the [calendar](https://fredhutch.github.io/FHBig/calendar/).
```
Comment (k8hertweck) [inline _drafts/community-group-overview.md]: Do we need to say "check the calendar" for EVERY group, or can we list that once at the top?

**Review by k8hertweck — COMMENTED** [review]
What about the Shiny and Cromwell groups?

File: _drafts/community-group-overview.md (hunk 1 above)
Comment (Chilliwack) [reply _drafts/community-group-overview.md]: added them

File: _drafts/community-group-overview.md (hunk 2 above)
Comment (Chilliwack) [reply _drafts/community-group-overview.md]: adding and flushed out how to start one section

File: _drafts/community-group-overview.md (hunk 3 above)
Comment (Chilliwack) [reply _drafts/community-group-overview.md]: good idea....I incorporated your wording

File: _drafts/community-group-overview.md (hunk 4 above)
Comment (Chilliwack) [reply _drafts/community-group-overview.md]: removed the multiple mentions and reference it once at the top and again in the Coop Monthly meeting blurb just bc that event changes locations sometimes...

**Pull request merged**